src/components/MultiHopTrade/hooks/quoteValidation/useInsufficientBalanceProtocolFeeMeta.tsx

```diff
@@ -21,8 +24,8 @@ import { useAppSelector } from 'state/store'
 export const useInsufficientBalanceProtocolFeeMeta = () => {
   const wallet = useWallet().state.wallet
 
-  const sellAssetAccountId = useAppSelector(selectSellAccountId)
-  const buyAssetAccountId = useAppSelector(selectBuyAccountId)
+  const sellAssetAccountId = useAppSelector(selectFirstHopSellAccountId)
```

Comment:
Wondering whether or not it would make sense to keep the old vernacular/implementation for the time being, and either return it (it being e.g `selectSellAccountId`), or `selectFirstHopSellAccountId` in case the flag is on?
Which should effectively be the same as what we're doing here, but would alleviate the risks associated to consuming the new implementation (missing occurences, mostly).

Though that would effectively be the same process done in two more steps, and the implementation looks sane, so I believe pending ops and runtime testing, we might just want to get this in as-is.

Reply:
Yeah ideally if we can check it's working as-is we should get it in to reduce the amount of cleanup we need to come back to